### PR TITLE
Refining TB1EPI definition

### PR DESCRIPTION
**Pull request by ChristophePhillips**
Better definition of what the TB1EPI sequence is.
Defining necessary metadata needed.

### CODE REVIEW

File: src/99-appendices/10-qmri.md [hunk 1]
```diff
@@ -243,12 +243,12 @@ maps are stored in the `fmap` folder.
...
+| Double-angle B1<sup>+</sup> mapping                    | TB1DAM   | RF Grouping | Groups images for B1<sup>+</sup> field mapping ([Insko and Bolinger 1993](https://www.sciencedirect.com/science/article/abs/pii/S1064185883711332)). Double angle method is based on the calculation of the actual angles from signal ratios, collected by two acquisitions at different nominal excitation flip angles. Common sequence types for this application include spin echo and echo planar imaging. _Associated output suffixes_: `TB1map`                                                                                                                                                      |
```
Comment (agahkarakuzu) [inline src/99-appendices/10-qmri.md:246]: I like the idea of `B1plusmap` --> `TB1map` both shorter and in better harmony with the RF grouping suffixes.

File: src/99-appendices/10-qmri.md [hunk 2]
```diff
@@ -272,9 +272,8 @@ The use of `fa` follows the default convention. However, this suffix defines a
 specific use case for the `echo` entity:
 
 At each `FlipAngle`, the `TB1EPI` suffix lists two images acquired at two echo times.
-However, the second echo in this method is generated in a different fashion compared 
-to the `MESE`. The first echo is a typical spin echo (SE), whereas the second
-echo is a stimulated echo (STE) that is not generated by an inversion pulse. 
+However, the second echo in this method is generated in a different fashion compared to the MESE.  Whereas the first echo is a typical spin echo (SE) formed from the pulses alpha-2alpha, the second echo is a stimulated echo (STE), formed by an additional alpha pulse (i.e. alpha-2alpha-alpha).
```
Comment (agahkarakuzu) [inline src/99-appendices/10-qmri.md:275]: > However, the second echo in this method is generated in a different fashion compared to the MESE.  Whereas the first echo is a typical spin echo (SE) formed from the pulses alpha-2alpha, the second echo is a stimulated echo (STE), formed by an additional alpha pulse (i.e. alpha-2alpha-alpha).

Reordering and rewording transitions for clarity: 

The first echo is a spin echo (SE) formed by the pulses alpha-2alpha. However, the second echo in this method is generated in a different fashion compared to a typical MESE acquisition. The second echo is a stimulated echo (STE) that is formed by an additional alpha pulse (i.e., alpha-2alpha-alpha).

File: src/99-appendices/10-qmri.md [hunk 3]
```diff
@@ -284,6 +283,11 @@ entity MUST index the images as follows:
 |Lower `EchoTime` | Higher `EchoTime`                  |
 |Spin Echo (SE) image|Stimulated Echo (STE) image|
 
+Two further parameters MUST also be defined in the JSON file
```
Comment (agahkarakuzu) [inline src/99-appendices/10-qmri.md:286]: > Two further parameters MUST also be defined in the JSON file

The following metadata fields MUST be defined in the accompanying JSON files:

**Comment by agahkarakuzu** [comment]
Resolve merge conflicts after #35 ready to be merged upon @ChristophePhillips approval re my suggestions.

**Comment by ChristophePhillips** [comment]
Happy with rewording! 😄

**Comment by agahkarakuzu** [comment]
Resolved by #38.

**Pull request closed without merging**